### DISCUSSION TITLE
Foocus Installation Problem

### DISCUSSION THREAD

**Discussion by Bdseo247** [lllyasviel/Fooocus#3716] [Q&A] [answered]
Hello, When I try to install fooocus in my local host it says **Press any key to continue.**  As I watch many videos for this problem but I can't solve it. Please help me. 
![press any key](https://github.com/user-attachments/assets/c380ac55-750f-4616-9c12-a8f202d72ee6)

**Comment by ghost** [comment]
Install nvidia drivers as it says.

**Comment by poor7** [comment, marked as answer]
It looks like it doesn't see your video card. If you have AMD, use the AMD manual. https://github.com/lllyasviel/Fooocus#windows-amd-gpus
